recipes/json-schema-validator/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+    "20200415":
```

Comment:
This project does releases. The last version is 2.0.0 and one year old.

And they plan new releases too https://github.com/pboettch/json-schema-validator/issues/80

Reply:
Their own Connafile uses version numbers too. So please add 2.0.0 or a pre-release, based on this number: https://github.com/pboettch/json-schema-validator/blob/2cc7e9aaa501938efa6b648d0f3561c4657ee9ca/CMakeLists.txt#L4

Reply:
Sadly the release does not include cmake support, I am also hoping for a fix to make it upstream.

I may mark this a draft while waiting